package-lock.json

```diff
@@ -102,9 +102,9 @@
             "dev": true
         },
         "@types/vscode": {
-            "version": "1.41.0",
-            "resolved": "https://registry.npmjs.org/@types/vscode/-/vscode-1.41.0.tgz",
-            "integrity": "sha512-7SfeY5u9jgiELwxyLB3z7l6l/GbN9CqpCQGkcRlB7tKRFBxzbz2PoBfGrLxI1vRfUCIq5+hg5vtDHExwq5j3+A==",
+            "version": "1.42.0",
```

Comment:
is this change intentional?

Reply:
Yes, codicon support for hover markdowns is introduced in 1.42. See zeplinHoverUtil.ts  in this pr

Reply:
then should this change be in package.json too?

Reply:
in package json it is marked with ^, so it is not necessary

Reply:
I changed the version to 1.42.0 via this command:
npm install @types/vscode@1.42.0 --save-dev --save-exact